sys-boot/tboot/tboot-1.10.3.ebuild

```diff
@@ -0,0 +1,79 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
```

Comment:
Can we do EAPI=8 now?

Reply:
mount-boot.eclass is not ported to EAPI 8.

Reply:
Well it apparently IS now so i updated it to EAPI=8